[2016-08-07 15:07:04] request.INFO: Matched route "{route}". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://localhost:8888/FLP/Symfony/web/","method":"GET"} []
[2016-08-07 15:07:05] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2016-08-07 15:07:05] request.CRITICAL: Uncaught PHP Exception Twig_Error_Syntax: "The block 'title' has already been defined line 7 in "base.html.twig" at line 7." at /Applications/MAMP/htdocs/FLP/Symfony/vendor/twig/twig/lib/Twig/TokenParser/Block.php line 31 {"exception":"[object] (Twig_Error_Syntax(code: 0): The block 'title' has already been defined line 7 in \"base.html.twig\" at line 7. at /Applications/MAMP/htdocs/FLP/Symfony/vendor/twig/twig/lib/Twig/TokenParser/Block.php:31)"} []
[2016-08-09 16:56:45] request.INFO: Matched route "{route}". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://localhost:8888/FLP/Symfony/web/","method":"GET"} []
[2016-08-09 16:56:46] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2016-08-09 16:56:46] request.CRITICAL: Uncaught PHP Exception Twig_Error_Syntax: "The block 'title' has already been defined line 7 in "base.html.twig" at line 7." at /Applications/MAMP/htdocs/FLP/Symfony/vendor/twig/twig/lib/Twig/TokenParser/Block.php line 31 {"exception":"[object] (Twig_Error_Syntax(code: 0): The block 'title' has already been defined line 7 in \"base.html.twig\" at line 7. at /Applications/MAMP/htdocs/FLP/Symfony/vendor/twig/twig/lib/Twig/TokenParser/Block.php:31)"} []
[2016-08-09 18:54:55] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /js/bootstrap.min.js" (from "http://localhost:8888/FLP/Symfony/web/app_dev.php/administrateur/liste/photo")" at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php line 2774 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /js/bootstrap.min.js\" (from \"http://localhost:8888/FLP/Symfony/web/app_dev.php/administrateur/liste/photo\") at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php:2774, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-08-09 18:54:55] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /css/bootstrap.min.css" (from "http://localhost:8888/FLP/Symfony/web/app_dev.php/administrateur/liste/photo")" at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php line 2774 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /css/bootstrap.min.css\" (from \"http://localhost:8888/FLP/Symfony/web/app_dev.php/administrateur/liste/photo\") at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php:2774, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-08-09 18:54:57] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /js/bootstrap.min.js" (from "http://localhost:8888/FLP/Symfony/web/app_dev.php/administrateur/liste/photo")" at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php line 2774 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /js/bootstrap.min.js\" (from \"http://localhost:8888/FLP/Symfony/web/app_dev.php/administrateur/liste/photo\") at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php:2774, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-08-09 18:55:37] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /css/bootstrap.min.css" (from "http://localhost:8888/FLP/Symfony/web/app_dev.php/administrateur/liste/photo")" at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php line 2774 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /css/bootstrap.min.css\" (from \"http://localhost:8888/FLP/Symfony/web/app_dev.php/administrateur/liste/photo\") at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php:2774, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-08-09 18:55:43] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /css/bootstrap.min.css" (from "http://localhost:8888/FLP/Symfony/web/app_dev.php/administrateur/liste/photo")" at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php line 2774 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /css/bootstrap.min.css\" (from \"http://localhost:8888/FLP/Symfony/web/app_dev.php/administrateur/liste/photo\") at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php:2774, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-08-09 18:55:55] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /css/bootstrap.min.css" (from "http://localhost:8888/FLP/Symfony/web/app_dev.php/administrateur/liste/photo")" at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php line 2774 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /css/bootstrap.min.css\" (from \"http://localhost:8888/FLP/Symfony/web/app_dev.php/administrateur/liste/photo\") at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php:2774, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-08-09 19:19:19] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /web/bundles/framework/images/logo_symfony.png"" at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php line 2774 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /web/bundles/framework/images/logo_symfony.png\" at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php:2774, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-08-09 19:31:02] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /js/nav.js" (from "http://localhost:8888/FLP/Symfony/web/app_dev.php/administrateur/liste/photo")" at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php line 2774 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /js/nav.js\" (from \"http://localhost:8888/FLP/Symfony/web/app_dev.php/administrateur/liste/photo\") at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php:2774, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-08-09 19:33:07] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /js/nav.js" (from "http://localhost:8888/FLP/Symfony/web/app_dev.php/administrateur/liste/photo")" at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php line 2774 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /js/nav.js\" (from \"http://localhost:8888/FLP/Symfony/web/app_dev.php/administrateur/liste/photo\") at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php:2774, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-08-13 16:47:49] request.INFO: Matched route "{route}". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://localhost:8888/FLP/Symfony/web/","method":"GET"} []
[2016-08-13 16:47:50] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2016-08-13 16:47:50] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "Class "AppBundle\Controller\DefaultController" does not exist." at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php line 2896 {"exception":"[object] (InvalidArgumentException(code: 0): Class \"AppBundle\\Controller\\DefaultController\" does not exist. at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php:2896)"} []
[2016-08-13 16:48:30] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /web/bundles/framework/images/logo_symfony.png"" at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php line 2774 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /web/bundles/framework/images/logo_symfony.png\" at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php:2774, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-08-14 14:58:30] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /dev.php/administrateur/liste/photo"" at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php line 2774 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /dev.php/administrateur/liste/photo\" at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php:2774, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-09-20 19:10:53] request.INFO: Matched route "{route}". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://localhost:8888/FLP/Symfony/web/","method":"GET"} []
[2016-09-20 19:10:53] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2016-09-20 19:10:53] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "Class "AppBundle\Controller\DefaultController" does not exist." at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php line 2896 {"exception":"[object] (InvalidArgumentException(code: 0): Class \"AppBundle\\Controller\\DefaultController\" does not exist. at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php:2896)"} []
[2016-09-20 19:11:06] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /web/bundles/framework/images/logo_symfony.png"" at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php line 2774 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /web/bundles/framework/images/logo_symfony.png\" at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php:2774, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-09-27 21:13:14] request.INFO: Matched route "{route}". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://localhost:8888/FLP/Symfony/web/","method":"GET"} []
[2016-09-27 21:13:14] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2016-09-27 21:13:14] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "Class "AppBundle\Controller\DefaultController" does not exist." at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php line 2896 {"exception":"[object] (InvalidArgumentException(code: 0): Class \"AppBundle\\Controller\\DefaultController\" does not exist. at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php:2896)"} []
[2016-09-27 21:13:31] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /web/bundles/framework/images/logo_symfony.png"" at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php line 2774 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /web/bundles/framework/images/logo_symfony.png\" at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php:2774, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-09-28 20:25:06] request.INFO: Matched route "{route}". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://localhost:8888/FLP/Symfony/web/","method":"GET"} []
[2016-09-28 20:25:06] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2016-09-28 20:25:06] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "Class "AppBundle\Controller\DefaultController" does not exist." at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php line 2896 {"exception":"[object] (InvalidArgumentException(code: 0): Class \"AppBundle\\Controller\\DefaultController\" does not exist. at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php:2896)"} []
[2016-10-08 14:09:17] request.INFO: Matched route "{route}". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://localhost:8888/FLP/Symfony/web/","method":"GET"} []
[2016-10-08 14:09:17] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2016-10-08 14:09:17] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "Class "AppBundle\Controller\DefaultController" does not exist." at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php line 2896 {"exception":"[object] (InvalidArgumentException(code: 0): Class \"AppBundle\\Controller\\DefaultController\" does not exist. at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php:2896)"} []
[2016-10-08 14:09:35] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /web/bundles/framework/images/logo_symfony.png"" at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php line 2774 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /web/bundles/framework/images/logo_symfony.png\" at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php:2774, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-10-10 21:31:44] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /bundles/Presentation/css/presentation.css" (from "http://localhost:8888/FLP/Symfony/web/app_dev.php/administrateur/presentation")" at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php line 2774 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /bundles/Presentation/css/presentation.css\" (from \"http://localhost:8888/FLP/Symfony/web/app_dev.php/administrateur/presentation\") at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php:2774, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-10-19 21:55:12] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /bundles/Portfolio/css/portfolioModificationBo.css" (from "http://localhost:8888/FLP/Symfony/web/app_dev.php/administrateur/portfolio/modification")" at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php line 2774 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /bundles/Portfolio/css/portfolioModificationBo.css\" (from \"http://localhost:8888/FLP/Symfony/web/app_dev.php/administrateur/portfolio/modification\") at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php:2774, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-10-19 21:55:20] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /bundles/Portfolio/css/portfolioModificationBo.css" (from "http://localhost:8888/FLP/Symfony/web/app_dev.php/administrateur/portfolio/modification")" at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php line 2774 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /bundles/Portfolio/css/portfolioModificationBo.css\" (from \"http://localhost:8888/FLP/Symfony/web/app_dev.php/administrateur/portfolio/modification\") at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php:2774, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-10-19 21:55:31] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /bundles/Portfolio/css/portfolioModificationBo.css" (from "http://localhost:8888/FLP/Symfony/web/app_dev.php/administrateur/portfolio/modification")" at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php line 2774 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /bundles/Portfolio/css/portfolioModificationBo.css\" (from \"http://localhost:8888/FLP/Symfony/web/app_dev.php/administrateur/portfolio/modification\") at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php:2774, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-10-25 21:14:03] request.INFO: Matched route "{route}". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://localhost:8888/FLP/Symfony/web/","method":"GET"} []
[2016-10-25 21:14:03] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2016-10-25 21:14:03] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "Class "AppBundle\Controller\DefaultController" does not exist." at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php line 2896 {"exception":"[object] (InvalidArgumentException(code: 0): Class \"AppBundle\\Controller\\DefaultController\" does not exist. at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php:2896)"} []
[2016-10-25 21:14:11] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /app_dev"" at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php line 2774 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /app_dev\" at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php:2774, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-10-25 21:14:15] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /app_dev/"" at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php line 2774 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /app_dev/\" at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php:2774, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-10-25 21:14:26] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /web/bundles/framework/images/logo_symfony.png"" at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php line 2774 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /web/bundles/framework/images/logo_symfony.png\" at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php:2774, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-10-25 23:08:34] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /path/to/css/fileinput.min.css" (from "http://localhost:8888/FLP/Symfony/web/app_dev.php")" at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php line 2774 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /path/to/css/fileinput.min.css\" (from \"http://localhost:8888/FLP/Symfony/web/app_dev.php\") at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php:2774, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-10-25 23:08:34] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /js/bootstrap-filestyle.min.js" (from "http://localhost:8888/FLP/Symfony/web/app_dev.php")" at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php line 2774 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /js/bootstrap-filestyle.min.js\" (from \"http://localhost:8888/FLP/Symfony/web/app_dev.php\") at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php:2774, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-10-25 23:08:34] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /path/to/font-awesome/css/font-awesome.min.css" (from "http://localhost:8888/FLP/Symfony/web/app_dev.php")" at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php line 2774 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /path/to/font-awesome/css/font-awesome.min.css\" (from \"http://localhost:8888/FLP/Symfony/web/app_dev.php\") at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php:2774, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-10-25 23:08:34] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /js/bootstrap-filestyle.min.js" (from "http://localhost:8888/FLP/Symfony/web/app_dev.php")" at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php line 2774 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /js/bootstrap-filestyle.min.js\" (from \"http://localhost:8888/FLP/Symfony/web/app_dev.php\") at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php:2774, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-10-26 17:32:27] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /path/to/css/fileinput.min.css" (from "http://localhost:8888/FLP/Symfony/web/app_dev.php")" at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php line 2774 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /path/to/css/fileinput.min.css\" (from \"http://localhost:8888/FLP/Symfony/web/app_dev.php\") at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php:2774, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-10-26 17:32:27] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /js/bootstrap-filestyle.min.js" (from "http://localhost:8888/FLP/Symfony/web/app_dev.php")" at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php line 2774 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /js/bootstrap-filestyle.min.js\" (from \"http://localhost:8888/FLP/Symfony/web/app_dev.php\") at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php:2774, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-10-26 17:32:27] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /path/to/font-awesome/css/font-awesome.min.css" (from "http://localhost:8888/FLP/Symfony/web/app_dev.php")" at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php line 2774 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /path/to/font-awesome/css/font-awesome.min.css\" (from \"http://localhost:8888/FLP/Symfony/web/app_dev.php\") at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php:2774, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-10-29 18:49:56] request.INFO: Matched route "{route}". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://localhost:8888/FLP/Symfony/web/","method":"GET"} []
[2016-10-29 18:49:56] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2016-10-29 18:49:56] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "Class "AppBundle\Controller\DefaultController" does not exist." at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php line 2896 {"exception":"[object] (InvalidArgumentException(code: 0): Class \"AppBundle\\Controller\\DefaultController\" does not exist. at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php:2896)"} []
[2016-10-29 18:50:12] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /path/to/font-awesome/css/font-awesome.min.css" (from "http://localhost:8888/FLP/Symfony/web/app_dev.php")" at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php line 2774 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /path/to/font-awesome/css/font-awesome.min.css\" (from \"http://localhost:8888/FLP/Symfony/web/app_dev.php\") at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php:2774, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-10-29 18:50:12] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /path/to/css/fileinput.min.css" (from "http://localhost:8888/FLP/Symfony/web/app_dev.php")" at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php line 2774 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /path/to/css/fileinput.min.css\" (from \"http://localhost:8888/FLP/Symfony/web/app_dev.php\") at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php:2774, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-10-29 18:50:12] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /js/bootstrap-filestyle.min.js" (from "http://localhost:8888/FLP/Symfony/web/app_dev.php")" at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php line 2774 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /js/bootstrap-filestyle.min.js\" (from \"http://localhost:8888/FLP/Symfony/web/app_dev.php\") at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php:2774, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-10-29 19:14:59] request.INFO: Matched route "{route}". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://localhost:8888/FLP/Symfony/web/","method":"GET"} []
[2016-10-29 19:14:59] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2016-10-29 19:14:59] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "Class "AppBundle\Controller\DefaultController" does not exist." at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php line 2896 {"exception":"[object] (InvalidArgumentException(code: 0): Class \"AppBundle\\Controller\\DefaultController\" does not exist. at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php:2896)"} []
[2016-10-29 19:43:34] request.INFO: Matched route "{route}". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://localhost:8888/FLP/Symfony/web/","method":"GET"} []
[2016-10-29 19:43:34] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2016-10-29 19:43:34] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "Class "AppBundle\Controller\DefaultController" does not exist." at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php line 2896 {"exception":"[object] (InvalidArgumentException(code: 0): Class \"AppBundle\\Controller\\DefaultController\" does not exist. at /Applications/MAMP/htdocs/FLP/Symfony/var/cache/prod/classes.php:2896)"} []
[2016-11-02 22:29:42] request.INFO: Matched route "{route}". {"route":"accueil_platform_homepage","route_parameters":{"_controller":"Accueil\\PlatformBundle\\Controller\\AccueilController::frontAction","_route":"accueil_platform_homepage"},"request_uri":"http://fleurdelysphotography.fr/web/","method":"GET"} []
[2016-11-02 22:29:42] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\DriverException: "An exception occured in driver: SQLSTATE[HY000] [2013] Lost connection to MySQL server at 'reading initial communication packet', system error: 111" at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 115 {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occured in driver: SQLSTATE[HY000] [2013] Lost connection to MySQL server at 'reading initial communication packet', system error: 111 at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:115, Doctrine\\DBAL\\Driver\\PDOException(code: 2013): SQLSTATE[HY000] [2013] Lost connection to MySQL server at 'reading initial communication packet', system error: 111 at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 2013): SQLSTATE[HY000] [2013] Lost connection to MySQL server at 'reading initial communication packet', system error: 111 at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43)"} []
[2016-11-02 22:29:42] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:29:42] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:29:42] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:29:42] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:29:42] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:29:42] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:29:42] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:29:42] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:29:42] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.finish_request","listener":"Symfony\\Component\\HttpKernel\\EventListener\\LocaleListener::onKernelFinishRequest"} []
[2016-11-02 22:29:42] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.finish_request","listener":"Symfony\\Component\\HttpKernel\\EventListener\\TranslatorListener::onKernelFinishRequest"} []
[2016-11-02 22:29:42] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.finish_request","listener":"Symfony\\Component\\HttpKernel\\EventListener\\RouterListener::onKernelFinishRequest"} []
[2016-11-02 22:29:42] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.finish_request","listener":"Symfony\\Component\\Security\\Http\\Firewall::onKernelFinishRequest"} []
[2016-11-02 22:29:42] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occured in driver: SQLSTATE[HY000] [2013] Lost connection to MySQL server at 'reading initial communication packet', system error: 111 at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 115) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occured in driver: SQLSTATE[HY000] [2013] Lost connection to MySQL server at 'reading initial communication packet', system error: 111 at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:115, Doctrine\\DBAL\\Driver\\PDOException(code: 2013): SQLSTATE[HY000] [2013] Lost connection to MySQL server at 'reading initial communication packet', system error: 111 at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 2013): SQLSTATE[HY000] [2013] Lost connection to MySQL server at 'reading initial communication packet', system error: 111 at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43)"} []
[2016-11-02 22:29:42] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\DriverException: "An exception occured in driver: SQLSTATE[HY000] [2013] Lost connection to MySQL server at 'reading initial communication packet', system error: 111" at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 115 {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occured in driver: SQLSTATE[HY000] [2013] Lost connection to MySQL server at 'reading initial communication packet', system error: 111 at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:115, Doctrine\\DBAL\\Driver\\PDOException(code: 2013): SQLSTATE[HY000] [2013] Lost connection to MySQL server at 'reading initial communication packet', system error: 111 at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 2013): SQLSTATE[HY000] [2013] Lost connection to MySQL server at 'reading initial communication packet', system error: 111 at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43, Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occured in driver: SQLSTATE[HY000] [2013] Lost connection to MySQL server at 'reading initial communication packet', system error: 111 at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:115, Doctrine\\DBAL\\Driver\\PDOException(code: 2013): SQLSTATE[HY000] [2013] Lost connection to MySQL server at 'reading initial communication packet', system error: 111 at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 2013): SQLSTATE[HY000] [2013] Lost connection to MySQL server at 'reading initial communication packet', system error: 111 at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43)"} []
[2016-11-02 22:29:42] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:29:42] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:29:42] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:29:42] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:29:42] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:29:42] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:29:42] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:29:42] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:29:42] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.finish_request","listener":"Symfony\\Component\\HttpKernel\\EventListener\\LocaleListener::onKernelFinishRequest"} []
[2016-11-02 22:29:42] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.finish_request","listener":"Symfony\\Component\\HttpKernel\\EventListener\\TranslatorListener::onKernelFinishRequest"} []
[2016-11-02 22:29:42] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.finish_request","listener":"Symfony\\Component\\HttpKernel\\EventListener\\RouterListener::onKernelFinishRequest"} []
[2016-11-02 22:29:42] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.finish_request","listener":"Symfony\\Component\\Security\\Http\\Firewall::onKernelFinishRequest"} []
[2016-11-02 22:29:42] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occured in driver: SQLSTATE[HY000] [2013] Lost connection to MySQL server at 'reading initial communication packet', system error: 111 at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 115) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occured in driver: SQLSTATE[HY000] [2013] Lost connection to MySQL server at 'reading initial communication packet', system error: 111 at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:115, Doctrine\\DBAL\\Driver\\PDOException(code: 2013): SQLSTATE[HY000] [2013] Lost connection to MySQL server at 'reading initial communication packet', system error: 111 at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 2013): SQLSTATE[HY000] [2013] Lost connection to MySQL server at 'reading initial communication packet', system error: 111 at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43)"} []
[2016-11-02 22:30:04] request.INFO: Matched route "{route}". {"route":"accueil_platform_homepage","route_parameters":{"_controller":"Accueil\\PlatformBundle\\Controller\\AccueilController::frontAction","_route":"accueil_platform_homepage"},"request_uri":"http://fleurdelysphotography.fr/web/","method":"GET"} []
[2016-11-02 22:30:04] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\DriverException: "An exception occured in driver: SQLSTATE[HY000] [2013] Lost connection to MySQL server at 'reading initial communication packet', system error: 111" at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 115 {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occured in driver: SQLSTATE[HY000] [2013] Lost connection to MySQL server at 'reading initial communication packet', system error: 111 at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:115, Doctrine\\DBAL\\Driver\\PDOException(code: 2013): SQLSTATE[HY000] [2013] Lost connection to MySQL server at 'reading initial communication packet', system error: 111 at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 2013): SQLSTATE[HY000] [2013] Lost connection to MySQL server at 'reading initial communication packet', system error: 111 at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43)"} []
[2016-11-02 22:30:04] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:30:04] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:30:04] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:30:04] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:30:04] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:30:04] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:30:04] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:30:04] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:30:04] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.finish_request","listener":"Symfony\\Component\\HttpKernel\\EventListener\\LocaleListener::onKernelFinishRequest"} []
[2016-11-02 22:30:04] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.finish_request","listener":"Symfony\\Component\\HttpKernel\\EventListener\\TranslatorListener::onKernelFinishRequest"} []
[2016-11-02 22:30:04] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.finish_request","listener":"Symfony\\Component\\HttpKernel\\EventListener\\RouterListener::onKernelFinishRequest"} []
[2016-11-02 22:30:04] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.finish_request","listener":"Symfony\\Component\\Security\\Http\\Firewall::onKernelFinishRequest"} []
[2016-11-02 22:30:04] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occured in driver: SQLSTATE[HY000] [2013] Lost connection to MySQL server at 'reading initial communication packet', system error: 111 at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 115) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occured in driver: SQLSTATE[HY000] [2013] Lost connection to MySQL server at 'reading initial communication packet', system error: 111 at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:115, Doctrine\\DBAL\\Driver\\PDOException(code: 2013): SQLSTATE[HY000] [2013] Lost connection to MySQL server at 'reading initial communication packet', system error: 111 at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 2013): SQLSTATE[HY000] [2013] Lost connection to MySQL server at 'reading initial communication packet', system error: 111 at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43)"} []
[2016-11-02 22:30:04] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\DriverException: "An exception occured in driver: SQLSTATE[HY000] [2013] Lost connection to MySQL server at 'reading initial communication packet', system error: 111" at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 115 {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occured in driver: SQLSTATE[HY000] [2013] Lost connection to MySQL server at 'reading initial communication packet', system error: 111 at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:115, Doctrine\\DBAL\\Driver\\PDOException(code: 2013): SQLSTATE[HY000] [2013] Lost connection to MySQL server at 'reading initial communication packet', system error: 111 at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 2013): SQLSTATE[HY000] [2013] Lost connection to MySQL server at 'reading initial communication packet', system error: 111 at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43, Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occured in driver: SQLSTATE[HY000] [2013] Lost connection to MySQL server at 'reading initial communication packet', system error: 111 at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:115, Doctrine\\DBAL\\Driver\\PDOException(code: 2013): SQLSTATE[HY000] [2013] Lost connection to MySQL server at 'reading initial communication packet', system error: 111 at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 2013): SQLSTATE[HY000] [2013] Lost connection to MySQL server at 'reading initial communication packet', system error: 111 at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43)"} []
[2016-11-02 22:30:04] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:30:04] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:30:04] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:30:04] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:30:04] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:30:04] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:30:04] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:30:04] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:30:04] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.finish_request","listener":"Symfony\\Component\\HttpKernel\\EventListener\\LocaleListener::onKernelFinishRequest"} []
[2016-11-02 22:30:04] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.finish_request","listener":"Symfony\\Component\\HttpKernel\\EventListener\\TranslatorListener::onKernelFinishRequest"} []
[2016-11-02 22:30:04] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.finish_request","listener":"Symfony\\Component\\HttpKernel\\EventListener\\RouterListener::onKernelFinishRequest"} []
[2016-11-02 22:30:04] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.finish_request","listener":"Symfony\\Component\\Security\\Http\\Firewall::onKernelFinishRequest"} []
[2016-11-02 22:30:04] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occured in driver: SQLSTATE[HY000] [2013] Lost connection to MySQL server at 'reading initial communication packet', system error: 111 at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 115) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occured in driver: SQLSTATE[HY000] [2013] Lost connection to MySQL server at 'reading initial communication packet', system error: 111 at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:115, Doctrine\\DBAL\\Driver\\PDOException(code: 2013): SQLSTATE[HY000] [2013] Lost connection to MySQL server at 'reading initial communication packet', system error: 111 at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 2013): SQLSTATE[HY000] [2013] Lost connection to MySQL server at 'reading initial communication packet', system error: 111 at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43)"} []
[2016-11-02 22:51:19] request.INFO: Matched route "{route}". {"route":"accueil_platform_homepage","route_parameters":{"_controller":"Accueil\\PlatformBundle\\Controller\\AccueilController::frontAction","_route":"accueil_platform_homepage"},"request_uri":"http://fleurdelysphotography.fr/web/","method":"GET"} []
[2016-11-02 22:51:49] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\DriverException: "An exception occured in driver: SQLSTATE[HY000] [2003] Can't connect to MySQL server on 'fleurdelbnanis.mysql.db' (4)" at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 115 {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occured in driver: SQLSTATE[HY000] [2003] Can't connect to MySQL server on 'fleurdelbnanis.mysql.db' (4) at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:115, Doctrine\\DBAL\\Driver\\PDOException(code: 2003): SQLSTATE[HY000] [2003] Can't connect to MySQL server on 'fleurdelbnanis.mysql.db' (4) at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 2003): SQLSTATE[HY000] [2003] Can't connect to MySQL server on 'fleurdelbnanis.mysql.db' (4) at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43)"} []
[2016-11-02 22:51:49] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:51:49] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:51:49] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:51:49] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:51:49] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:51:49] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:51:49] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:51:49] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:52:19] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.finish_request","listener":"Symfony\\Component\\HttpKernel\\EventListener\\LocaleListener::onKernelFinishRequest"} []
[2016-11-02 22:52:19] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.finish_request","listener":"Symfony\\Component\\HttpKernel\\EventListener\\TranslatorListener::onKernelFinishRequest"} []
[2016-11-02 22:52:19] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.finish_request","listener":"Symfony\\Component\\HttpKernel\\EventListener\\RouterListener::onKernelFinishRequest"} []
[2016-11-02 22:52:19] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.finish_request","listener":"Symfony\\Component\\Security\\Http\\Firewall::onKernelFinishRequest"} []
[2016-11-02 22:52:19] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occured in driver: SQLSTATE[HY000] [2003] Can't connect to MySQL server on 'fleurdelbnanis.mysql.db' (4) at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 115) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occured in driver: SQLSTATE[HY000] [2003] Can't connect to MySQL server on 'fleurdelbnanis.mysql.db' (4) at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:115, Doctrine\\DBAL\\Driver\\PDOException(code: 2003): SQLSTATE[HY000] [2003] Can't connect to MySQL server on 'fleurdelbnanis.mysql.db' (4) at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 2003): SQLSTATE[HY000] [2003] Can't connect to MySQL server on 'fleurdelbnanis.mysql.db' (4) at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43)"} []
[2016-11-02 22:52:19] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\DriverException: "An exception occured in driver: SQLSTATE[HY000] [2003] Can't connect to MySQL server on 'fleurdelbnanis.mysql.db' (4)" at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 115 {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occured in driver: SQLSTATE[HY000] [2003] Can't connect to MySQL server on 'fleurdelbnanis.mysql.db' (4) at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:115, Doctrine\\DBAL\\Driver\\PDOException(code: 2003): SQLSTATE[HY000] [2003] Can't connect to MySQL server on 'fleurdelbnanis.mysql.db' (4) at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 2003): SQLSTATE[HY000] [2003] Can't connect to MySQL server on 'fleurdelbnanis.mysql.db' (4) at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43, Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occured in driver: SQLSTATE[HY000] [2003] Can't connect to MySQL server on 'fleurdelbnanis.mysql.db' (4) at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:115, Doctrine\\DBAL\\Driver\\PDOException(code: 2003): SQLSTATE[HY000] [2003] Can't connect to MySQL server on 'fleurdelbnanis.mysql.db' (4) at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 2003): SQLSTATE[HY000] [2003] Can't connect to MySQL server on 'fleurdelbnanis.mysql.db' (4) at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43)"} []
[2016-11-02 22:52:19] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:52:19] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:52:19] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:52:19] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:52:19] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:52:19] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:52:19] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:52:19] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:52:10] request.INFO: Matched route "{route}". {"route":"accueil_platform_homepage","route_parameters":{"_controller":"Accueil\\PlatformBundle\\Controller\\AccueilController::frontAction","_route":"accueil_platform_homepage"},"request_uri":"http://fleurdelysphotography.fr/web/","method":"GET"} []
[2016-11-02 22:52:40] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\DriverException: "An exception occured in driver: SQLSTATE[HY000] [2003] Can't connect to MySQL server on 'fleurdelbnanis.mysql.db' (4)" at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 115 {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occured in driver: SQLSTATE[HY000] [2003] Can't connect to MySQL server on 'fleurdelbnanis.mysql.db' (4) at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:115, Doctrine\\DBAL\\Driver\\PDOException(code: 2003): SQLSTATE[HY000] [2003] Can't connect to MySQL server on 'fleurdelbnanis.mysql.db' (4) at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 2003): SQLSTATE[HY000] [2003] Can't connect to MySQL server on 'fleurdelbnanis.mysql.db' (4) at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43)"} []
[2016-11-02 22:52:40] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:52:40] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:52:40] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:52:40] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:52:40] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:52:40] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:52:40] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:52:40] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:52:49] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.finish_request","listener":"Symfony\\Component\\HttpKernel\\EventListener\\LocaleListener::onKernelFinishRequest"} []
[2016-11-02 22:52:49] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.finish_request","listener":"Symfony\\Component\\HttpKernel\\EventListener\\TranslatorListener::onKernelFinishRequest"} []
[2016-11-02 22:52:49] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.finish_request","listener":"Symfony\\Component\\HttpKernel\\EventListener\\RouterListener::onKernelFinishRequest"} []
[2016-11-02 22:52:49] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.finish_request","listener":"Symfony\\Component\\Security\\Http\\Firewall::onKernelFinishRequest"} []
[2016-11-02 22:52:49] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occured in driver: SQLSTATE[HY000] [2003] Can't connect to MySQL server on 'fleurdelbnanis.mysql.db' (4) at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 115) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occured in driver: SQLSTATE[HY000] [2003] Can't connect to MySQL server on 'fleurdelbnanis.mysql.db' (4) at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:115, Doctrine\\DBAL\\Driver\\PDOException(code: 2003): SQLSTATE[HY000] [2003] Can't connect to MySQL server on 'fleurdelbnanis.mysql.db' (4) at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 2003): SQLSTATE[HY000] [2003] Can't connect to MySQL server on 'fleurdelbnanis.mysql.db' (4) at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43)"} []
[2016-11-02 22:53:10] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.finish_request","listener":"Symfony\\Component\\HttpKernel\\EventListener\\LocaleListener::onKernelFinishRequest"} []
[2016-11-02 22:53:10] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.finish_request","listener":"Symfony\\Component\\HttpKernel\\EventListener\\TranslatorListener::onKernelFinishRequest"} []
[2016-11-02 22:53:10] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.finish_request","listener":"Symfony\\Component\\HttpKernel\\EventListener\\RouterListener::onKernelFinishRequest"} []
[2016-11-02 22:53:10] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.finish_request","listener":"Symfony\\Component\\Security\\Http\\Firewall::onKernelFinishRequest"} []
[2016-11-02 22:53:10] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occured in driver: SQLSTATE[HY000] [2003] Can't connect to MySQL server on 'fleurdelbnanis.mysql.db' (4) at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 115) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occured in driver: SQLSTATE[HY000] [2003] Can't connect to MySQL server on 'fleurdelbnanis.mysql.db' (4) at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:115, Doctrine\\DBAL\\Driver\\PDOException(code: 2003): SQLSTATE[HY000] [2003] Can't connect to MySQL server on 'fleurdelbnanis.mysql.db' (4) at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 2003): SQLSTATE[HY000] [2003] Can't connect to MySQL server on 'fleurdelbnanis.mysql.db' (4) at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43)"} []
[2016-11-02 22:53:10] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\DriverException: "An exception occured in driver: SQLSTATE[HY000] [2003] Can't connect to MySQL server on 'fleurdelbnanis.mysql.db' (4)" at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 115 {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occured in driver: SQLSTATE[HY000] [2003] Can't connect to MySQL server on 'fleurdelbnanis.mysql.db' (4) at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:115, Doctrine\\DBAL\\Driver\\PDOException(code: 2003): SQLSTATE[HY000] [2003] Can't connect to MySQL server on 'fleurdelbnanis.mysql.db' (4) at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 2003): SQLSTATE[HY000] [2003] Can't connect to MySQL server on 'fleurdelbnanis.mysql.db' (4) at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43, Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occured in driver: SQLSTATE[HY000] [2003] Can't connect to MySQL server on 'fleurdelbnanis.mysql.db' (4) at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:115, Doctrine\\DBAL\\Driver\\PDOException(code: 2003): SQLSTATE[HY000] [2003] Can't connect to MySQL server on 'fleurdelbnanis.mysql.db' (4) at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 2003): SQLSTATE[HY000] [2003] Can't connect to MySQL server on 'fleurdelbnanis.mysql.db' (4) at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43)"} []
[2016-11-02 22:53:10] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:53:10] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:53:10] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:53:10] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:53:10] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:53:10] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:53:10] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:53:10] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.request","listener":"Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener::__invoke"} []
[2016-11-02 22:53:40] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.finish_request","listener":"Symfony\\Component\\HttpKernel\\EventListener\\LocaleListener::onKernelFinishRequest"} []
[2016-11-02 22:53:40] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.finish_request","listener":"Symfony\\Component\\HttpKernel\\EventListener\\TranslatorListener::onKernelFinishRequest"} []
[2016-11-02 22:53:40] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.finish_request","listener":"Symfony\\Component\\HttpKernel\\EventListener\\RouterListener::onKernelFinishRequest"} []
[2016-11-02 22:53:40] event.DEBUG: Notified event "{event}" to listener "{listener}". {"event":"kernel.finish_request","listener":"Symfony\\Component\\Security\\Http\\Firewall::onKernelFinishRequest"} []
[2016-11-02 22:53:40] request.CRITICAL: Exception thrown when handling an exception (Doctrine\DBAL\Exception\DriverException: An exception occured in driver: SQLSTATE[HY000] [2003] Can't connect to MySQL server on 'fleurdelbnanis.mysql.db' (4) at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 115) {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occured in driver: SQLSTATE[HY000] [2003] Can't connect to MySQL server on 'fleurdelbnanis.mysql.db' (4) at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:115, Doctrine\\DBAL\\Driver\\PDOException(code: 2003): SQLSTATE[HY000] [2003] Can't connect to MySQL server on 'fleurdelbnanis.mysql.db' (4) at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 2003): SQLSTATE[HY000] [2003] Can't connect to MySQL server on 'fleurdelbnanis.mysql.db' (4) at /home/fleurdelbn/www/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43)"} []
